app/views/measures/_guidance_table.html.erb

```diff
@@ -0,0 +1,31 @@
+<details class="govuk-details" data-module="govuk-details">
```

Comment:
We'll have to rename this file if we change the component type from a table to a summary list, say. How do you feel about just calling this partial `_guidance.html.erb?`

Reply:
_measure_condition_guidances.html.erb